framework/intercepts.cpp

```diff
@@ -21,6 +21,19 @@ void __declspec(naked) _chatInput() {
 			ret
 	}
 }
+void __fastcall keyPressEvent(WPARAM wparam, LPARAM lparam);
+void __declspec(naked) _keyPressIntercept() {
+	__asm {
+		
+		mov cl, [edi + 0x08] //(wparam)

```

Comment:
I think wparam and lparam are int, not byte, so you'll want to use CX and DX instead of CL and DL.
CL = (count lower, 8), CH (count higher, 8), CX (count, 16), ECX (extended count, 32)